scalableminds/webknossos
# update husky for faster post-checkout

[Husky](https://typicode.github.io/husky/) is a githooks tool installed via yarn. This PR updates it from 0.14.3 to 9.1.5 with the main benefit being speed (no need to go via `yarn run` anymore).

For me, git checkout with no changes now takes 20–30ms, rather than the 230–270ms before this change.

### Steps to test:
- Check out this branch locally, run yarn install, run `time git checkout update-husky`, should be fast.
